kiwi/package_manager/apt.py

```diff
@@ -145,28 +145,24 @@ def process_install_requests_bootstrap(self):
                 options=['-a', '-H', '-X', '-A']
             )
         except Exception as e:
-            Path.wipe(bootstrap_dir)
             raise KiwiDebootstrapError(
                 '%s: %s' % (type(e).__name__, format(e))
             )
-        chroot_apt_get_args = self.root_bind.move_to_root(
-            self.apt_get_args
-        )
-        bash_command = [
-            'rm', '-r', '-f', bootstrap_dir, '&&',
-            'chroot', self.root_dir, 'apt-get'
-        ] + chroot_apt_get_args + self.custom_args + [
-            'update'
-        ]
-        Command.run(
-            ['bash', '-c', ' '.join(bash_command)], self.command_env
-        )
+        finally:
+            Path.wipe(bootstrap_dir)
+
         return self.process_install_requests()
 
     def process_install_requests(self):
         """
         Process package install requests for image phase (chroot)
         """
+        update_command = ['chroot', self.root_dir, 'apt-get']
+        update_command.extend(self.root_bind.move_to_root(self.apt_get_args))
+        update_command.extend(self.custom_args)
+        update_command.append('update')
+        Command.run(update_command, self.command_env)
+
```

Comment:
Ok, I agree this can be sorted in a separate commit. About the motivation, apt doesn't have an automatic refresh system as it happens in zypper, dnf or yum. So, if the boostrap is not executed it turns that packages are not installable because the debian repository database is not populated yet.

Maybe I missed something but I believe this is the same behavior as we have with zypper using `--refresh,` a repository database update is run before every zypper install. I checked that `apt-get update` doesn't re-download data if it is already up to date, like zypper does.